src/sentry/api/endpoints/organization_events.py

```diff
@@ -53,6 +55,7 @@
 class OrganizationEventsV2Endpoint(OrganizationEventsV2EndpointBase):
     """Deprecated in favour of OrganizationEventsEndpoint"""
 
+    @deprecated(datetime.fromisoformat("2022-07-21T00:00:00+00:00:00"), suggested_api="events")
```

Comment:
I'd either go with the Docs page title: "Query Discover Events in Table Format"

Or the route template: `api/0/organizations/{organization_slug}/events/`